src/test/scala/nl.knaw.dans.easy.bag2deposit/AppSpec.scala

```diff
@@ -32,9 +32,8 @@ class AppSpec extends AnyFlatSpec with Matchers with AppConfigSupport with FileS
     }
     new EasyConvertBagToDespositApp(appConfig).addPropsToBags(
       (testDir / "exports").children,
-      IdType.DOI,
       None,
-      DepositPropertiesFactory(appConfig)
+      DepositPropertiesFactory(appConfig, IdType.DOI, BagSource.VAULT)
```

Comment:
After we remove the SIP-part from `easy-fedora-to-bag` we need some tests here for BagSource.FEDORA as well.

Reply:
The option results in not-implemented anyway so far, or don't we need further chages here?

Reply:
if it comes from FEDORA 
* it shouldn't consult the bag-index,
* the sword token should be empty, or non-existent
* the bag-id is empty
* TYPE should be DOI
* anything else?